aspnetcore/security/authentication/social/google-logins.md

```diff
@@ -51,7 +52,7 @@ Store sensitive settings such as the Google client ID and secret values with [Se
 
 You can manage your API credentials and usage in the [API Console](https://console.developers.google.com/apis/dashboard).
 
-## Configure Google authentication
+## Configure Google authentication (First Edition)
```

Comment:
If `(First Edition)` is deprecated, delete it. Just list the latest instructions and delete everything deprecated.

Reply:
I edited

Reply:
thank you for help